cli: Fix output encoding

rvl opened the pull request:
This avoids the error

    <stdout>: hPutChar: invalid argument (invalid character)

when the user does not have a UTF-8 locale set.

KtorZ commented on src/Cardano/Wallet/API/V1/Types.hs:
```diff
@@ -1138,6 +1101,73 @@ instance ToHttpApiData AccountIndex where
+-- | A type modelling the request for a new 'EosWallet',
+-- on the mobile client or hardware wallet.
+data NewEosWallet = NewEosWallet
+    { neweoswalAccountsPublicKeysWithIxs :: ![AccountPublicKeyWithIx]
```
Remember that we are making an API for human to use :upside_down_face:  .. `accountPublicKeys` was fine, or simply, "accounts".

denisshevchenko replied:
Agree. Fixed.

KtorZ commented:
@denisshevchenko hlint isn't happy with this one ^^^^

denisshevchenko commented:
@KtorZ Thanks, fixed.

denisshevchenko commented:
> account indexes are not used for anything in EOS wallet so far, am I right?

@akegalj We're storing accounts for EOS-wallets in the same hierarchy, I mean `HdAccount`, and it contains `HdAccountId` which includes `HdAccountIx`. :-)

rvl committed:
cli: Fix output encoding

This avoids the error

    <stdout>: hPutChar: invalid argument (invalid character)

when the user does not have a UTF-8 locale set.

KtorZ reviewed (approved):
Nice.
